scadalts-ui/src/store/alarms/notifications.js

```diff
@@ -160,6 +160,49 @@ const storeAlarmsNotifications = {
 			}
 			return { edId, ehId: eventHandler.id };
 		},
+
+		async createDoubleEventHandler({state, dispatch}, payload) {
```

Comment:
I would suggest Dual instead of Double, the latter name is used to denote the numeric type, and this context has nothing to do with it.